xapian-glib/xapian-mset-iterator.cc

```diff
@@ -288,16 +285,21 @@ xapian_mset_iterator_new (XapianMSet *mset)
  * xapian_mset_iterator_is_valid:
  * @iter: a #XapianMSetIterator
  *
- * Checks whether @iter is valid.
+ * Prior to 1.4, this function would check whether @iter was valid.  However
+ * #XapianMSetIterator objects are now always valid, and this function always
+ * returns %TRUE.
+ *
+ * Returns: %TRUE
  *
- * Returns: %TRUE if the iterator is valid
+ * Deprecated: 1.4: The iterator is now always valid, so checks using this
```

Comment:
The symbol in the header file should also be annotated with `XAPIAN_GLIB_DEPRECATED_IN_1_6`. I've just added the macros in Git master.